pkg/clients/imagebuilder/client.go

```diff
@@ -67,8 +67,9 @@ type OSTree struct {
 
 // Customizations is made of the packages that are baked into an image
 type Customizations struct {
-	Packages            *[]string     `json:"packages"`
-	PayloadRepositories *[]Repository `json:"payload_repositories,omitempty"`
+	Packages            *[]string                `json:"packages"`
+	PayloadRepositories *[]Repository            `json:"payload_repositories,omitempty"`
+	Subscription        models.ImageSubscription `json:"subscription"`
```

Comment:
subscription is not mandatory in  Customizations and should be the same as image builder 

```go


```
```suggestion
	Subscription        *models.ImageSubscription `json:"subscription,omitempty"`
```